docs/dev_guide.rst

```diff
@@ -0,0 +1,116 @@
+.. _dev_guide:
+
+*****************
+Developer's Guide
+*****************
+
+Developers who wish to contribute to this package can fork the main `GitHub repository <https://github.com/Helioviewer-Project/python-api>`__.
+
+.. note::
+    When you install the dependencies, you need to install the development version of the package.
+    See the :ref:`obtaining_the_source` section for details.
+
+Building Documentation
+----------------------
+
+.. note::
+
+    Building the documentation is not necessary unless you are
+    writing new documentation or do not have internet access, because
+    the latest versions of ``hvpy``'s documentation should be available
+    at `hvpy.readthedocs.io <https://hvpy.readthedocs.io/>`__.
```

Comment:
```suggestion
    Building the documentation is not necessary unless you are writing new documentation or do not have internet access, because the latest versions of ``hvpy``'s should be available at `hvpy.readthedocs.io <https://hvpy.readthedocs.io/>`__.
```
Should I do it this way for a long sentence.

Reply:
Yes.